apps/client/src/pages/CheckerPage.js

```diff
@@ -70,7 +71,7 @@ const CheckerPage = ({ checker, topic, resetChecker }) => {
   };
 
   const isFinished = (component) => isActive(component, true);
-
+  matomoPageView();
```

Comment:
This looks very weird IMO. Is this not re-rendering all the time? Should we not wrap it in a `UseEffect()`?